app/src/main/java/com/capstone/event_finder/activities/MainActivity.java

```diff
@@ -68,6 +62,10 @@ private void setUpBottomNavigation(FragmentManager fragmentManager) {
         bottomNavigationView.setSelectedItemId(R.id.action_feed);
     }
 
+    private void makeInitialCallGetEvents() {
+        eventViewModel.refreshEvents();
+    }
+
```

Comment:
✅ Looks good as far as `MainActivity` is concerned!  

This helps clean up the direct communication between the activity and fragment, which is generally bad practice.